public/js/fxa-analytics.js

```diff
@@ -211,6 +212,8 @@ function setGAListeners(){
     ga("create", "UA-77033033-16");
     ga("set", "anonymizeIp", true);
     ga("set", "dimension6", `${document.body.dataset.signedInUser}`);
+
+    // Growth Experiment
     if (document.body.dataset.experiment) {
```

Comment:
Unreachable Code – This is gated on the ENV variable, `EXPERIMENT_ACTIVE` set to 1.